explorer/src/controllers/admin/heads.ts

```diff
@@ -11,10 +10,8 @@ const router = Router()
 
 router.get('/heads', async (req, res) => {
   const params = parseParams(req.query)
-  const db = await getDb()
-  const ethereumHeadRepository = getCustomRepository(
+  const ethereumHeadRepository = getConnection().getCustomRepository(
```

Comment:
```suggestion
  const ethereumHeadRepository = getCustomRepository(
```

Reply:
As a side note, im thinking that we never even needed custom repositories, could have used regular repositories all along with their default methods unless we use custom logic in them